components/ListView/ComponentInputs.js

```diff
@@ -2,133 +2,155 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import ComponentTypeIcons from '../../components/ListView/ComponentTypeIcons';
 
-
 class ComponentInputs extends React.Component {
-
   componentDidMount() {
-    this.bindPopover();
-    this.bindViewDetails();
     this.initializeBootstrapElements();
+    this.bindTooltipShow();
+    this.bindHideTooltip();
+    this.bindTooltipMouseleave();
   }
 
   componentDidUpdate() {
     this.unbind();
-    this.bindPopover();
-    this.bindViewDetails();
     this.initializeBootstrapElements();
-    $('#cmpsr-recipe-inputs .list-group-item').popover('destroy');
+    this.bindTooltipShow();
+    this.bindHideTooltip();
+    this.bindTooltipMouseleave();
+    this.hideTooltip('all');
   }
 
   componentWillUnmount() {
     this.unbind();
+    this.hideTooltip('all');
   }
 
-  bindPopover() {
-    // click input to view popover
-    $('#cmpsr-recipe-inputs').on('click', '.list-group-item', (event) => {
-      if (!$(event.target).is('a, a>span')) {
-        if ($(this).is('[aria-describedby]')) {
-          $(this).popover('destroy');
-        } else {
-          $('#cmpsr-recipe-inputs .list-group-item').popover('destroy');
-          $(this).popover('show');
-        }
+  bindTooltipShow() {
+    $('.cmpsr-list-inputs').off().on('mouseenter focus', '[data-toggle="tooltip"]', event => {
+      // prevent li tooltip from flashing when focus moves to the <a>
+      event.stopPropagation();
+      // hide tooltip for other list items
+      if ($(event.currentTarget).hasClass('list-pf-container')) {
+        $('.list-pf-container[data-toggle="tooltip"]').not(event.target).tooltip('hide');
+      }
+      // hide tooltip for component list item if hovering over an action
+      if ($(event.currentTarget).parent('.list-pf-actions').length) {
+        this.hideTooltip('parent');
       }
+      $(event.currentTarget).tooltip('show');
+      console.log('show', event.currentTarget);
```

Comment:
Could you please remove this line of debug code? This debug code will generate console output in browser.